scripts/globals/chocobo.lua

```diff
@@ -81,6 +81,45 @@ xi.chocobo.initZone = function(zone)
     end
 end
 
+xi.chocobo.renterOnTrade = function(player, npc, trade, eventSucceed, eventFail)
+    local zoneId = player:getZoneID()
+    local info   = chocoboInfo[zoneId]
```

Comment:
You check for `info` being nil in `renterOnEventFinish`, but not here

Reply:
Done.